src/oc_erchef/apps/oc_chef_wm/src/oc_chef_wm_sup.erl

```diff
@@ -98,19 +98,22 @@ maybe_start_action(false, Workers) ->
 
 
 check_actions_queue_at_capacity(PoolNameAtom, Vhost, Queue) ->
-    PreventStartupOnCap = ?QUEUE_MONITOR_SETTING(prevent_erchef_startup_on_full_capacity, false),
+    %% TODO(ssd) 2020-08-10: This whole function is now silly. This fixes dialyzer, but
+    %% really the whole thing can be removed, but really really all of the rabbitmq
+    %% management code can be removed SOON (TM).
+    %% PreventStartupOnCap = ?QUEUE_MONITOR_SETTING(prevent_erchef_startup_on_full_capacity, false),
     {MaxLength, CurrentLength, QueueAtCapacity} =
         chef_wm_rabbitmq_management:sync_check_queue_at_capacity(PoolNameAtom, Vhost, Queue),
     case QueueAtCapacity of
-        true ->
-            case PreventStartupOnCap of
-                true ->
-                    lager:critical("Vhost ~p, queue ~p at capacity, cannot start", [Vhost, Queue]),
-                    erlang:error(analytics_queue_at_capacity);
-                false ->
-                    lager:warning("Vhost ~p, queue ~p at capacity", [Vhost, Queue]),
-                    {MaxLength, CurrentLength}
-            end;
+        %% true ->
+        %%     case PreventStartupOnCap of
+        %%         true ->
+        %%             lager:critical("Vhost ~p, queue ~p at capacity, cannot start", [Vhost, Queue]),
+        %%             erlang:error(analytics_queue_at_capacity);
+        %%         false ->
+        %%             lager:warning("Vhost ~p, queue ~p at capacity", [Vhost, Queue]),
+        %%             {MaxLength, CurrentLength}
+        %%     end;
```

Comment:
Again, this was in rseponse to the function above always returning false, but that might not be true if we don't comment out some of the code.

Reply:
Fixed.